ocw/lib/gce.py

```diff
@@ -10,6 +10,8 @@
 
 class GCE(Provider):
     __instances = {}
+    # XXX: Perhaps we should get this from pcw.ini
```

Comment:
1. please delete the comment  , no point to keep it in the code 
2. I would say that from beginning it is fine to keep it hardcoded . If there will be need to tweak it we will analyze this need and decide about moving it to pcw.ini

Reply:
The XXX was added for discussion.

Fixed.